dss-api.yml

```diff
@@ -1449,6 +1449,70 @@ paths:
                     enum: [unhandled_exception, Forbidden, Unauthorized, OAuthResponseProblem, not_found, read_only]
                 required:
                   - code
+  /bundles:
+    get:
+      operationId: dss.api.bundles.enumerate
+      summary: List through all bundles available to the data-store
+      description: >
+        Given a prefix
+      parameters:
+        - name: replica
+          in: query
+          description: Replica to fetch from.
+          required: true
+          type: string
+          enum: [aws, gcp]
+        - name: prefix
+          in: query
+          description: used to search for keys under a specific "bundle/" prefix
+          required: false
+          type: string
+        - name: token
+          in: query
+          description: Token to manage retries.  End users constructing queries should not set this parameter.
+          required: false
+          type: string
+        - name: search_after
+          in: query
+          description: >
+            **Search-After-Context**.
+            An internal state pointer parameter for use with pagination. This parameter is used as a reference to what information the previous page has, and what needs to be returned next. This value should not be set by the user.
```

Comment:
Please use the same description provided in the POST /search endpoint.

Reply:
complete